kpet/data.py

```diff
@@ -530,7 +560,23 @@ def get_sets_from_regex_list(regex_list):
                 if not case.sets <= suite.sets:
                     raise Invalid("Case sets are not a subset of suite sets "
                                   "in suite: {}\ncase: {}".
-                                  format(suite.description, case.name))
+                                  format(suite.name, case.name))
+
+    def validate_suite_ids(self):
```

Comment:
This code is basically a duplicate of validate_case_ids(). I'd probably like to see these merged.

Reply:
Yeah, I thought about making a function returning a set of repeated members in a list, for example, but it wouldn't make this code much shorter. Do you still want me to get the common code out of these?

Reply:
I'll leave that to you.